src/allmydata/node.py

```diff
@@ -67,7 +67,7 @@ def _common_valid_config():
 
 # Add our application versions to the data that Foolscap's LogPublisher
 # reports.
-for thing, things_version in get_package_versions().iteritems():
+for thing, things_version in get_package_versions().items():
```

Comment:
I chose to use `items` here because the dictionary here is expected to be small (100s or less), so safe to consume greedily on Python 2. One could wrap in `six.iteritems`, but that seems unnecessary debt to incur here.